import-brage/src/test/java/no/unit/nva/importbrage/XmlImportTest.java

```diff
@@ -15,8 +16,14 @@
 
     @Test
     void xmlImportLoadsData() throws IOException {
-        var contributors = XmlImport.map(new File(getClass().getResource(OVERCOMPLETE_EXAMPLE).getFile()));
-        contributors.forEach(this::testContributor);
+        var publication = XmlImport.map(new File(getClass().getResource(OVERCOMPLETE_EXAMPLE).getFile()));
```

Comment:
Break this test into multiple tests. The tests could fail for more than one reasons. Also values are not verified to much the expected values. I would suggest to create a smaller file where you test that the values are read correctly (and they are not for example empty strings).